lib/compilers/sass.js

```diff
@@ -3,8 +3,8 @@ var path = require('path')
 var ensureRequire = require('../ensure-require.js')
 
 module.exports = function (raw, cb, compiler, filePath) {
-  ensureRequire('sass', 'node-sass')
-  var sass = require('node-sass')
+  ensureRequire('sass', 'sass')
```

Comment:
I don't know if this is actually necessary, but I didn't want to rewrite too much right now.